### PR TITLE
Replace dbplyr::in_schema with DBI::Id; Update renv version

### PR DESCRIPTION
**Pull request by brancengregory**
This used to not be possible, replace `dbplyr::in_schema` with `DBI::Id`, but there must have been a change to `tbl`

### CODE REVIEW

File: R/ojo_tbl.R
Comment (andrewjbe) [inline R/ojo_tbl.R:1]: Do we need to bump the version number for this?

**Review by andrewjbe — COMMENTED** [review]
Everything ran on my computer (I did `devtools::check()` as well) and looks good to me! Just want to make sure we don't need to bump the version number, etc.

File: R/ojo_tbl.R
Comment (brancengregory) [reply R/ojo_tbl.R:1]: We will bump on the main branch once merged, thanks

**Comment by brancengregory** [comment]
@andrewjbe This is ready for approval and merging please!

**Review by andrewjbe — APPROVED** [review]
Forgot to do this earlier, my bad!

**Pull request merged**